File: sample.c
#include "zzzjson.h"
#include <stdio.h>
int main()
{
    Allocator *A = NewAllocator();
    Value *v = NewValue(A);
    const char *json = "{\"key123\":123}";
    BOOL ret = ParseFast(v, json);
    if (ret == True)
    {
        Value *vv = ObjGet(v, "key123");
        SetKeyLenFast(vv, "key234", 6);
        const char *v_str = Stringify(v);
        if (v_str != 0) printf("%s\n", v_str);
    }
    ReleaseAllocator(A);
}

File: sample/go/sample.go
package main

import (
	"fmt"

	"github.com/dacez/zzzjson"
)

func main() {

	A := zzzjson.NewAllocator()
	defer A.ReleaseAllocator()
	V := zzzjson.NewValue(A)
	json := `[true,false]`

	ret := V.Parse(json)
	if ret != true {
		fmt.Println("Parse Error!")
		return
	}

	V1 := V.ArrayGet(1)
	ret = V1.SetStr("1234..456")
	if ret != true {
		fmt.Println("SetNumStr Error!")
		return
	}
	out := V.Stringify()
	if out != nil {
		fmt.Println(*out)
	}

	return
}
